plugins/destiny.py

```diff
@@ -422,13 +430,15 @@ def lore(text, bot):
     name = ''
     if not text:  # if we aren't searching, return a random card
         name = sample(list(LORE_CACHE), 1)[0]
+        if name in {'ghost_count', 'grim_tally'}:
```

Comment:
Hm, could also just sample 3 ("badwords" + 1) and shift the sampled list until you get a not-badword.